principles/fp-019-term-stability.md

```diff
@@ -0,0 +1,144 @@
+---
+layout: principle
+id: fp-019-term-stability
+title: Stability of Term Meaning (principle 19)
+---
+
+GO TO: [Recommendations/Requirements](#recommendations-and-requirements) &#124; [Implementation](#implementation) &#124; [Examples/Counter&#8209;Examples](#examples) &#124; [Criteria&nbsp;for&nbsp;Review](#criteria-for-review) &#124; [Feedback/Discussion](#feedback-and-discussion)
+
+Summary
+-------
+
+The definition of a term MUST always denote the same thing(s)--known as "referent(s)"--in reality. If a proposed change to the definition would substantially change its referents, then a new term with new IRI and definition MUST instead be created.
+
+Purpose
+-------
+
+Users of an ontology depend on the stability of its terms and their meanings. Therefore, changes to the definition of a term should never substantially shift its meaning. Put another way, its set of referents MUST remain stable, within reason. That is, changes to a term definition should not cause that term to point to different entities than it denotes already.
+
+Recommendations and Requirements
+-------
+
+If changing a term definition would change its referents, then instead a new term MUST be created with a new IRI and the new definition. Minor changes to the definition for clarity, grammar, and/or proper punctuation that do not change the referents are permitted. What is considered a 'minor change' will likely need to be considered on a case-by-case basis; it is left to the ontology developers to decide. However, any feedback from users MUST be taken into account.
+
+Conditions under which a term must be deprecated according to this principle, or for which term deprecation should be considered, include:
+
+1) The old textual definition misses its intended target. This includes cases where the term refers to non-existent referents (as might happen, for example, when new research reveals that the referent does not exist in reality).
+1) The original term definition is considered sufficiently “damaged” (too vague, too restrictive, too misused or too misunderstood).
+
+In all cases the developers SHOULD provide guidance on how to handle deprecated terms (either by exact replacement or by considering other terms), and be mindful of the potential costs to users of the ontology who might use the existing term. As well, developers SHOULD pre-announce term obsoletions. See [Principle 13](http://obofoundry.org/principles/fp-013-notification.html) for guidance on such announcements.
+
+Implementation
+-------
+
+Detailed procedures for obsoleting a term are described on the OBO Academy page [Obsoleting an Existing Ontology Term](https://oboacademy.github.io/obook/howto/obsolete-term/). 
```

Comment:
See also [kgcl:NodeObsoletion](http://w3id.org/kgcl/NodeObsoletion) for a taxonomy of kinds of obsoletion

Reply:
@cmungall So far as I can tell, IAO has basically the same set of reasons. The benefit to mentioning this other resource is unclear.